balena/cog-fdo

```diff
@@ -23,7 +23,7 @@ function launch
 	do
         killall /usr/bin/cog
         echo "Launching Cog for $@ ..."
-        /usr/bin/cog -P fdo $@
+        /usr/bin/cog -P wl $@
```

Comment:
If you want, it's possible to remove the `-P wl` because nowadays Cog autodetects which platform module it should usee.

Reply:
nice tip.